workspaces/ui/src/services/testing/TestingService.js

```diff
@@ -0,0 +1,3 @@
+"use strict";
```

Comment:
sigh...is there a way to automatically ignore these? I would have put them in a separate package in our workspace so it uses the same typescript pathway as (most of) the rest of the packages

Reply:
It shouldn't be building them in-place in the first place. Perhaps something your IDE is doing?